xresources/README.md

```diff
@@ -0,0 +1,5 @@
+# Xresources substrata theme
+
+If you want to use this theme, you can simply copy the content of `substrata.xresources` in your `~/.Xresources`. Otherwise, you can include it in your `~/.Xresources` by adding the following line to your configuration :
+
+`#include "/path/to/substrata.xresources"`
```

Comment:
Would it be ok if you removed this README, instead moving the text from here into the root project’s README?

Reply:
Nope, no stress at all, I'll fix this tonight ! :)